ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -255,6 +255,15 @@ export class EvalCommand extends Command<EvalCommandReturn> {
             ERR(`one or more depends of key "${key}" is missing a value`, { location: [`EvalCommand`, 'run'] }),
           );
         }
+
+        if (evaluatedValue && cfgu.aliases && !cfgu.aliases.some((alias) => alias === key)) {
```

Comment:
There are few things i don't understand here:
- Why you feel like `validate` is the right place for aliases? EvalCommand can run without validation if needed. it's in its props.
- This condition forces me to have one of my aliases as the name of the config. This is not one of the requirements from the issue.
- even if i'll do as this condition expect, i'd still won't have the aliases in the result from EvalCommand. see attached
```javascript
// Schema
schema = {
  "MY_EMAIL_ID": {
    "type": "String",
    "default": "1",
    "aliases": [
      "emailId", "MY_EMAIL_ID"
    ]
  }
}
// EvalCommand result
result = { MY_EMAIL_ID: '1' }

// Expected result:
result = { MY_EMAIL_ID: '1', emailId: '1' }
```

Reply:
removed it